### ISSUE TITLE
[BUG]Search Results Container Overflow Bug in Job Search Interface

### ISSUE THREAD

**Issue by Devmangrani** [Devmangrani/JobSewa#77]
**Describe the bug**
When searching for jobs (e.g., "Driver"), the search result cards are overflowing their designated container boxes. This creates a poor visual experience and affects the overall layout of the page. The profile image, salary information, and deadline details are not properly contained within their parent container.

**To Reproduce**

Go to the main job search page
Enter "Driver" in the search box
Click the "SEARCH" button or press Enter
Observe how the search result cards, particularly the profile image and job details, overflow beyond their container boundaries

**Expected behavior**
Search result cards should be properly contained within their designated boxes
Profile images should maintain their aspect ratio without overflowing
Salary and deadline information should be aligned properly within the card
Content should be responsive and adjust to different screen sizes
Cards should have consistent spacing and padding

**Screenshots**
<img width="1448" alt="Screenshot 2024-10-26 at 1 34 09 PM" src="https://github.com/user-attachments/assets/d0a384ce-03cd-4bef-8d32-48884580f616">

**Comment by Kehinde13** [comment]
Can you assign this to me

**Comment by Devmangrani** [comment]
I'll assign it to you—I'm looking forward to seeing your contribution.

**Comment by Kehinde13** [comment]
I keep getting "Sorry, no jobs found" when I try to reproduce the bug

**Comment by Devmangrani** [comment]
No, there isn’t a default username and password. You’ll need to create an account or use your existing credentials to log in. Let me know if you have any other questions!

**Comment by Kehinde13** [comment]
Thanks for the opportunity,
Can you assign one more issue to me?

**Comment by Devmangrani** [comment]
Thank you for your great work, @Kehinde13  I’ve recently posted two new issues, and there’s also the "Dark mode on whole website" issue if you’d like to work on that. Let me know if you're interested!
